rascaline-c-api/CMakeLists.txt

```diff
@@ -175,46 +155,88 @@ else()
     set(STRIP_ARGS "")
 endif()
 
+if (RASCALINE_BUILD_FOR_PYTHON)
+    # strip dynamic library for smaller wheels to download/install
+    set(CARGO_RUSTC_ARGS "${CARGO_RUSTC_ARGS};-Cstrip=symbols")
+endif()
+
 add_custom_target(cargo-build-rascaline ALL
     COMMAND ${CMAKE_COMMAND} -E env
         MACOSX_DEPLOYMENT_TARGET=${CMAKE_OSX_DEPLOYMENT_TARGET}
-        RUSTFLAGS="${RUST_FLAGS}"
+        RUSTFLAGS=${RUST_FLAGS}
         cargo rustc ${CARGO_BUILD_ARG} -- ${SONAME_ARGS} ${STRIP_ARGS}
     WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
     DEPENDS ${ALL_RUST_SOURCES}
     COMMENT "Building ${RASCALINE_LIB_NAME} with cargo"
+    VERBATIM
+)
+
+add_custom_command(TARGET cargo-build-rascaline
+    POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy ${RASCALINE_CAPI_LOCATION} ${RASCALINE_LOCATION}
+    DEPENDS cargo-build-rascaline
 )
 
 add_dependencies(rascaline cargo-build-rascaline)
 set(RASCALINE_HEADERS
-    "${PROJECT_SOURCE_DIR}/include/equistore.h"
     "${PROJECT_SOURCE_DIR}/include/rascaline.h"
-    "${PROJECT_SOURCE_DIR}/include/equistore.hpp"
     "${PROJECT_SOURCE_DIR}/include/rascaline.hpp"
 )
 set(RASCALINE_INCLUDE_DIR ${PROJECT_SOURCE_DIR}/include/)
 
+set(EQUISTORE_GIT_VERSION "6c5a833")
+if (RASCALINE_FETCH_EQUISTORE)
+    message(STATUS "Fetching equistore @ ${EQUISTORE_GIT_VERSION} from github")
+
+    if (CMAKE_VERSION VERSION_LESS 3.11)
+        # cmake 3.10 (default on ubuntu 20.04) does not include FetchContent,
+        # but we can use the version from cmake 3.11 without change
+        set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/fetch-content-cmake-3.11;${CMAKE_MODULE_PATH}")
+    endif()
+
+    include(FetchContent)
+    FetchContent_Declare(
+        equistore
+        URL      https://github.com/lab-cosmo/equistore/archive/${EQUISTORE_GIT_VERSION}.zip
+        URL_HASH MD5=5adcd26f236022ea89fa0287b83c6609
+        SOURCE_SUBDIR equistore-core
+        VERBOSE
+    )
+
+    if (CMAKE_VERSION VERSION_GREATER 3.18)
+        FetchContent_MakeAvailable(equistore)
+    else()
+        if (NOT equistore_POPULATED)
+            FetchContent_Populate(equistore)
+        endif()
+
+        add_subdirectory(${equistore_SOURCE_DIR}/equistore-core ${equistore_BINARY_DIR})
+    endif()
+
+    add_dependencies(cargo-build-rascaline equistore)
+else()
+    find_package(equistore 0.1 REQUIRED CONFIG)
```

Comment:
Is 0.1 the current equistore version? I thought we are still on 0.0?

Reply:
I updated to 0.1 in the C-API change